interface-elements-for-web/articles/rich-text-editor/tables.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Tables
+
+---
+# Tables
+You can create and edit tables in documents. 
+
+Click the **Table** button in **Insert** tab to insert a table. Then select the table size and click the corresponding square in the drop-down box.
```

Comment:
"click the corresponding square" What does 'corresponding' refer to?

Reply:
Changed to 'Click the **Table** button in the **Insert** tab and select the table size in the drop-down box.'